### PR TITLE
Add missing outfits

### PR DESCRIPTION
**Pull request by majkus522**

### CODE REVIEW

File: assets/data/characters/ayaka/en.json [hunk 1]
```diff
@@ -178,5 +178,15 @@
...
-  "weapon_type": "SWORD"
+  "weapon_type": "SWORD",
+  "outfits": [
+    {
+      "type": "Themed",
+      "name": "Springbloom Missive",
+      "description": "Ayaka's travel outfit. Its design is based on the Fontaine dresses depicted in light novel illustrations and seems to reflect the lifestyle of proper ladies who live in that nation.",
+      "rarity": 4,
+      "price": 1680,
+      "image": "outfit-springbloom-missive"
```
Comment (M4GICB) [inline assets/data/characters/ayaka/en.json]: [type should be Travel](https://genshin-impact.fandom.com/wiki/Springbloom_Missive)

File: assets/data/characters/jean/en.json [hunk 2]
```diff
@@ -180,6 +180,13 @@
       "rarity": 4,
       "price": 1680,
       "image": "outfit-sea-breeze-dandelion"
+    },
+    {
+      "type": "Alternate",
+      "name": "Gunnhildr's Legacy",
+      "description": "Jean's outfit. This classic knight's ceremonial outfit is suited to all manner of occasions.",
+      "rarity": 5,
+      "image": "outfit-gunnhildr-s-legacy"
```
Comment (M4GICB) [inline assets/data/characters/jean/en.json]: [rarity should be 4](https://genshin-impact.fandom.com/wiki/Gunnhildr%27s_Legacy)

File: assets/data/characters/mona/en.json [hunk 3]
```diff
@@ -178,5 +178,14 @@
...
   "vision_key": "HYDRO",
-  "weapon_type": "CATALYST"
+  "weapon_type": "CATALYST",
+  "outfits": [
+    {
+      "type": "Alternate",
+      "name": "Pact of Stars and Moon",
+      "description": "Mona's outfit. A costume made specifically to encompass the astrological concept of \"the destined moment,\" thanks to the 30% discount ticket she got from the planetarium refund.",
+      "rarity": 5,
+      "image": "outfit-pact-of-stars-and-moon"
```
Comment (M4GICB) [inline assets/data/characters/mona/en.json]: [rarity should be 4](https://genshin-impact.fandom.com/wiki/Pact_of_Stars_and_Moon)

**Review by M4GICB — CHANGES_REQUESTED** [review]
needs a few edits on rarities

**Comment by majkus522** [comment]
Fixed

**Review by M4GICB — APPROVED** [review]

**Pull request merged**